pipeline/metadata/flatten_satellite.py

```diff
@@ -75,111 +76,146 @@ def format_timestamp(timestamp: str) -> str:
   return f'{date}T{time}{timezone_hour}:{timezone_seconds}'
 
 
-def _process_received_ips(
-    row: Row, received_ips: Optional[Dict[str, str]]) -> Iterator[Row]:
+def _process_received_ips_v1(
+    row: Row, received_ips: Optional[Dict[str, List[str]]]) -> Iterator[Row]:
   """Add received_ip metadata to a Satellite row
+
   Args:
     row: existing row of satellite data
```

Comment:
Is `row` what we read from their file? What does it represent?

Reply:
`row` is the python object we are constructing in order to write it as a row in bigquery. Data read from the scan file is usually called `line` (if it's a json string) or `scan` if we've parsed the json into a python object.